### PR TITLE
Add delete all lessons of a person function

### PR DESCRIPTION
**Pull request by ethantyh692**

### CODE REVIEW

File: src/main/java/tutorease/address/model/Model.java [hunk 1]
```diff
@@ -115,4 +115,9 @@ public interface Model {
      * Returns size of lesson schedule.
      */
     int getLessonScheduleSize();
+
+    /**
+     * Deletes all the lessons of a particular student
+     */
```
Comment (NorbertLoh) [inline src/main/java/tutorease/address/model/Model.java:121]: perhaps right this in the javadoc form with the parameter.
```suggestions
/**
* Deletes all the lessons of a particular student
*
* @param student The person object to look up.
*/

**Review by NorbertLoh — COMMENTED** [review]
Nits to fix. Otherwise LGTM!

**Review by Chronoxy — CHANGES_REQUESTED** [review]
LGTM! However, some checks seems to be failing, can take a look and fix them before we merge

**Comment by ethantyh692** [comment]
> LGTM! However, some checks seems to be failing, can take a look and fix them before we merge

Haven't tested this new function yet so the checks are failing. Will create new issue for junit once merged.

**Comment by NorbertLoh** [comment]
Perhaps you can override it and just make it return null for now and implement it later. So that our checks do not let any possible issues go through for this PR.

**Comment by ethantyh692** [comment]
> Perhaps you can override it and just make it return null for now and implement it later. So that our checks do not let any possible issues go through for this PR.

@NorbertLoh @Chronoxy errors fixed

**Pull request merged**